### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Gautam2305**
- completed my landing page with categories and working buttons which navigate to wishlist, cart, product list and logout pages
- made three categories of men, women and footwear

https://user-images.githubusercontent.com/95231941/160239757-1e221cd1-5083-437b-8dce-5ca8a29ca204.mp4

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,45 @@
...
-  display: flex;
-}
-
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
-}
+@import url("https://sierra-shop.netlify.app/style.css");
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css");
```
Comment (kjais1720) [inline src/App.css]: import the css in the index.html, its better for performance, as we can use attributes like preconnect with it

**Review by SimranjitKaurMaan — APPROVED** [review]
You can move the images to cloudinary cdn and use the urls instead of keeping them in assets.

File: src/components/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import adidasShoes from "../assets/adidas-shoes.jpg";
```
Comment (SimranjitKaurMaan) [inline src/components/Category.jsx:1]: You can keep these urls in the backend in the model

Comment (SimranjitKaurMaan) [reply src/components/Category.jsx:1]: These are suggestions. You can proceed with merge for now.

File: src/components/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        <div>
+            <main className="main-login main-home">
+                <div className="category-header">
+                <h2 className="brand-heading">
+                    Women's Clothing
+                </h2 >
+                </div>
+                <div className="sale-img-container">
+                    <div>
+                        <Link to="/products" element={<ProductsListPage/>}>
```
Comment (kjais1720) [inline src/components/Category.jsx]: You don't need to specify the element in the `Link` component, it is specified in the route component.

File: src/components/Category.jsx (hunk 2 above)
Comment (Gautam2305) [reply src/components/Category.jsx:1]: Will do the cloudinary part in another branch separately as i progress further with this project....... thank you for your feedback. 🌻

File: src/App.css (hunk 1 above)
Comment (Gautam2305) [reply src/App.css]: The changes have been made. Please review them.

File: src/components/Category.jsx (hunk 3 above)
Comment (Gautam2305) [reply src/components/Category.jsx]: These changes have been taken care of. Please review the changes and approve them. Thank you for your feedback🌻

File: src/components/Category.jsx (hunk 2 above)
Comment (Gautam2305) [reply src/components/Category.jsx:1]: please approve the changes

Comment (SimranjitKaurMaan) [reply src/components/Category.jsx:1]: Already approved.

**Review by kjais1720 — APPROVED** [review]
All good now 👍

**Pull request merged**